### ISSUE TITLE
Ability to set a color for full slider bar without using CSS

### ISSUE THREAD

**Issue by rathodsanjay** [angular-slider/ng5-slider#77]
I assume, at present there is no way to change the color of the slider's full bar using options similar to `getTickColor` or `getSelectionBarColor`. Currently, it is possible to change it only by overriding the CSS classes related to the full bar element.

I am using ng5-slider in one of my Angular component and I want to change the color of the full bar dynamically - is there any way to do so?

Thanks.

![issue](https://user-images.githubusercontent.com/17136385/53937092-161fbf80-40d2-11e9-84c0-272301cc6144.png)

**Comment by piotrdz** [comment]
Correct. There is currently no way to change this programmatically. I can add this feature, but I'll need to think how to do that in sensible way.

**Comment by rathodsanjay** [comment]
Great. Thank you.

**Comment by JuliaRakitina** [comment]
I need this option too (

**Comment by fyitaaagmail** [comment]
Any news?
